examples/specialty_plots/ishikawa.py

```diff
@@ -0,0 +1,232 @@
+"""
+===============
+Ishikawa Diagrams
```

Comment:
The ==== lines need to be the same number of characters as the title "Ishikawa Diagrams"